openedx_events/management/commands/generate_avro_schemas.py

```diff
@@ -29,14 +29,14 @@ class Command(BaseCommand):
     Example::
 
         # one signal
-        python manage.py cms generate_avro_schemas org.openedx.learning.course.enrollment.changed.v1
```

Comment:
Updated docs here to remove `cms` as this will cause an error when running the management command.

Reply:
@rgraber: Just checking whether this was just a typo in the orginal docs, or whether this had significance that we need to look into. Thanks.

Reply:
Hmm. This is how you run the command in edx-platform but not anything else, because edx-platform requires you to specify lms or cms when running any management command. Maybe we should have 2 examples, one for running in edx-platform and one for running somewhere else?

Reply:
@rgraber: Actually, is there a reason why this needs to be a management command run from elsewhere, rather than just a script that could be run from openedx-events? I don't get why we'd want to involve any other repo in the creation of tests from this repo, but maybe that is required for some reason.

Reply:
It shouldn't require any other repo. We could move it to a standalone script.

Reply:
If it runs here, then all is well. I just didn’t understand the `cms`  comment. All good.